kubernetes/samples/scripts/create-weblogic-domain/manually-create-domain/domain.yaml

```diff
@@ -0,0 +1,106 @@
+  webLogicCredentialsSecret:
+    # Update this with the name of the secret containing your WebLogic server boot credentials:
+    name: domain1-weblogic-credentials
+
+  # If you want to include the server out file into the pod's stdout, set this to `true`:
+  includeServerOutInPodLog: true
+
+  # If you want to use a mounted volume as the log home, i.e. to persist logs outside the container, then
+  # uncomment this and set it to `true`:
+  # logHomeEnabled: false
+  # The in-pod name of the directory to store the domain, node manager, server logs, and server .out
+  # files in.
+  # If not specified or empty, domain log file, server logs, server out, and node manager log files
+  # will be stored in the default logHome location of /shared/logs/<domainUID>/.
+  # logHome: /shared/logs/domain1
+
+  # serverStartPolicy legal values are "NEVER", "IF_NEEDED", or "ADMIN_ONLY"
+  # This determines which WebLogic Servers the Operator will start up when it discovers this Domain
+  # - "NEVER" will not start any server in the domain
+  # - "ADMIN_ONLY" will start up only the administration server (no managed servers will be started)
+  # - "IF_NEEDED" will start all non-clustered servers, including the administration server and clustered servers up to the replica count
+  serverStartPolicy: "IF_NEEDED"
+
+  serverPod:
+    # an (optional) list of environment variable to be set on the servers
+    env:
+    - name: JAVA_OPTIONS
+      value: "-Dweblogic.StdoutDebugEnabled=false"
+    - name: USER_MEM_ARGS
+      value: "-Xms64m -Xmx256m "
+
+    # If you are storing your domain on a persistent volume (as opposed to inside the Docker image),
+    # then uncomment this section and provide the PVC details and mount path here (standard images
+    # from Oracle assume the mount path is `/shared`):
+    # volumes:
+    # - name: weblogic-domain-storage-volume
+    #   persistentVolumeClaim:
+    #     claimName: domain1-weblogic-sample-pvc
+    # volumeMounts:
+    # - mountPath: /shared
+    #   name: weblogic-domain-storage-volume
+
+  # adminServer is used to configure the desired behavior for starting the administration server.
+  adminServer:
+    # serverStartState legal values are "RUNNING" or "ADMIN"
+    # "RUNNING" means the listed server will be started up to "RUNNING" mode
+    # "ADMIN" means the listed server will be start up to "ADMIN" mode
+    serverStartState: "RUNNING"
+    adminService:
+      channels:
+       # Update this to set the NodePort to use for the Admin Server's default channel (where the
+       # admin console will be available):
+       - channelName: default
+         nodePort: 30701
+       # Uncomment to export the T3Channel as a service
+       #- channelName: T3Channel
+
+  # clusters is used to configure the desired behavior for starting member servers of a cluster.
+  # If you use this entry, then the rules will be applied to ALL servers that are members of the named clusters.
+  clusters:
+  - clusterName: cluster-1
```

Comment:
Add a comment saying something like "Update this with the name of your cluster. You can also add additional clusters"?